### PR TITLE
Minor changes to allow migration to ecmwf-lab

### PR DESCRIPTION
**Pull request by cvitolo**
Hello! Here are some minor changes to address Florian's comments and allow migration to ecmwf-lab.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,3 +124,8 @@ Notebooks for training and inference:
 - Match the features of the new dataset with the existing features. This can be done by going through `src/EDA/EDA_pre-processed_data.ipynb`.
 - Add the feature path as a variable to `src/utils/data_paths.py`. Further the path variable is needed to be added to either the time dependant or independant list (depending on which category it belongs to) present inside `export_feature_paths()`.
 - The model will now also be trained on the added feature while running src/train.py!
+
+## Info
+This repository was developed by Anurag Saha Roy (@lazyoracle) and Roshni Biswas (@roshni-b) for the ESA-SMOS-2020 project. The repository is now maintained by the Wildfire Danger Forecasting team at the European Centre for Medium-range Weather Forecast.
```
Comment (floriankrb) [inline README.md:129]: would you have a contact email to add there ?

Comment (cvitolo) [reply README.md:129]: Hi @floriankrb, is this question for Roshni/Anurag or for the team at ECMWF?

Comment (floriankrb) [reply README.md:129]: I don't know. All stakeholders should agree on one (or two) contact person, I assume.

**Comment by lazyoracle** [comment]
Addressed in latest code release.

**Pull request closed without merging**